src/cursor/find_cursor.ts

```diff
@@ -329,21 +329,6 @@ export class FindCursor<TSchema = Document> extends AbstractCursor<TSchema> {
     return this;
   }
 
-  /**
-   * Set a maxTimeMS on the cursor query, allowing for hard timeout limits on queries (Only supported on MongoDB 2.6 or higher)
-   *
-   * @param value - Number of milliseconds to wait before aborting the query.
-   */
-  maxTimeMS(value: number): this {
```

Comment:
does it matter that this used to be defined on `kBuiltOptions`, which gets spread during initialize on L73, and now it's deeper on the `cursorOptions`, which second in the spread? the comment there explicitly says that the order matters, i.e. that those cursorOptions take priority over the find cursor options, and, while that seems backwards, it's a little bit ominous, so I just wanted to know if you explicitly looked into that or have any context on the comment

Reply:
Ah kBuiltOptions vs kOptions, really good catch I did not catch the difference in the super class. I've reverted